defaults/test_default.yaml

```diff
@@ -233,3 +233,5 @@ use_dns_names: false
 use_placement_group: false
 
 scylla_d_overrides_files: []
+
+run_commit_log_check_thread: true
```

Comment:
by default we enable the thread, but the feature is enabled only on one test, wouldn't it gonna fail on any other test ?

Reply:
feature will be enabled by default Scylla https://github.com/scylladb/scylladb/pull/10974

iIdealy it should work in all tests. but this line is made to remove it by default, instead of reverting all code. or disable in some tests if needed 

scilla req: The commitlog didn't go over the size limit. It is a little more involved, we should make sure that the size of the commitlog directory never exceeds the limit, we can't rely on metrics since if there is a bug in the metrics we will never catch an actual bug where the directory is larger than the stats and the size limit was actually breached.